cfme/tests/cloud_infra_common/test_relationships.py

```diff
@@ -390,3 +396,72 @@ def test_change_network_security_groups_per_page_items():
         1524443
     """
     pass
+
+
+@test_requirements.relationships
+@pytest.mark.manual
+@pytest.mark.tier(1)
+@pytest.mark.meta(coverage=[1729953])
+def test_datastore_relationships():
+    """
+    Polarion:
+        assignee: ghubale
+        casecomponent: Cloud
+        caseimportance: medium
+        initialEstimate: 1/12h
+        tags: relationship
+        setup:
+            1. Add infrastructure provider(e.g: vsphere65)
+            2. Provision VM
+            3. Setup SSA
+        testSteps:
+            1. Navigate to VM's details page and click on 'Datastores' from 'Relationships' table
+            2. Click on 'Managed VMs' from 'Relationships' table
+            3. Select VM(Vm should be in 'on' state) and perform operations(Refresh Relationships
+               and Power States, Perform Smartstate Analysis, Extract Running Processes) by
+               selecting from 'configuration' dropdown
+        expectedResults:
+            1.
+            2.
+            3. Operations should be performed successfully. It should not give unexpected error.
+
+    Bugzilla:
+        1729953
+    """
+    pass
+
+
+@test_requirements.relationships
+@pytest.mark.manual
+@pytest.mark.tier(1)
+@pytest.mark.meta(coverage=[1732370])
+def test_cluster_relationships():
+    """
+    Polarion:
+        assignee: ghubale
+        casecomponent: Cloud
```

Comment:
**Required**  Casecomponent should be `Infra` as you are adding infra provider in step 1 below.